geode-core/src/main/java/org/apache/geode/management/internal/cli/result/AbstractResultData.java

```diff
@@ -147,11 +147,10 @@ public ResultData addAsFile(String fileName, String fileContents, String message
 
   public ResultData addAsFile(String fileName, byte[] data, int fileType, String message,
       boolean addTimeStampToName) {
-    byte[] bytes = data;
```

Comment:
I might by projecting my Python and C expectations into my relatively new Java experience, but I thought this assignment wouldn't change the mutability of `data`.  Wouldn't this need `byte[] bytes = data.clone();` to actually provided immutability to `data`?  Or is Java more defensive than I realized?